src/main/java/org/fundacionjala/coding/ariel/Katas2.java

```diff
@@ -0,0 +1,52 @@
+package org.fundacionjala.coding.ariel;
+
+import java.util.Arrays;
+
+import static java.util.Collections.reverseOrder;
+
+/**
+ * Kata2__Ariel Gonzales Vargas..
+ */
+public class Katas2 {
+    private String text;
+    private String[] textos;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the "textos" field and declare it as a local variable in the relevant methods. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1450)